CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to ADC
+
+All kinds of contributions are welcome! Whether it is:
+
+- Reporting a bug
+- Proposing a new feature
+- Improving documentation
+- Submitting a fix
+
+## Reporting Bugs and Suggesting New Features
+
+We use [GitHub issues](https://github.com/api7/adc/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) to track bugs and feature requests. Please make sure your requests are as detailed as possible to ensure minimum turnaround time.
+
+## Proposing Changes
+
+Pull requests are the best way to propose changes to ADC. Please follow these steps while submitting a pull request:
+
+1. Fork the repo and create your branch from main.
+2. If you've added code that should be tested, add tests.
+3. If you've changed behavior, update the [documentation](./README.md).
+4. Open that pull request!
+5. Ensure the test suites pass in CI.
+
+See [GitHub Flow](https://guides.github.com/introduction/flow/index.html).
+
+## Building ADC Locally
+
+To make changes to the code, you need to build ADC locally.
+
+First, you need to install [Go](https://go.dev/) in your development environment. ADC uses Go v1.20.
+
+Then fork and clone the repository, navigate to it in your machine and run:
+
+```shell
+make build
+```
+
+Now to test a particular command, for example `ping`, you can run:
```

Comment:
I think we should add a section on how to connect to APISIX.

Reply:
Good idea. Added how to deploy APISIX and connect to it.